packages/ts/react-crud/src/styles.css

```diff
@@ -0,0 +1,83 @@
+.auto-crud {
+  display: flex;
+}
+
+.auto-crud .auto-crud-main {
+  flex: 1 1 100%;
+  display: flex;
+  flex-direction: column;
+}
+
+.auto-crud .auto-crud-main .auto-crud-toolbar {
+  display: flex;
+  flex-shrink: 0;
+  align-items: baseline;
+  justify-content: flex-end;
+
+  padding: var(--lumo-space-s) var(--lumo-space-m);
+  background-color: var(--lumo-contrast-5pct);
+  border: 1px solid var(--lumo-contrast-10pct);
+  border-top: none;
+}
+
+.auto-crud .auto-form, .auto-crud-dialog .auto-form {
+  display: flex;
+  flex-direction: column;
+  align-items: stretch;
+}
+
+.auto-crud .auto-form {
+  width: 40%;
+  border: solid 1px var(--lumo-contrast-20pct);
+  border-left-width: 0;
+}
+
+.auto-crud .auto-form-fields, .auto-crud-dialog .auto-form-fields {
```

Comment:
Could we use more BEM approach here? There are too many nested classes here IMO.

Reply:
Simplified selectors where class names were redundant.